app/scripts/background.js

```diff
@@ -142,6 +143,24 @@ const PHISHING_WARNING_PAGE_TIMEOUT = ONE_SECOND_IN_MILLISECONDS;
 // Event emitter for state persistence
 export const statePersistenceEvents = new EventEmitter();
 
+if (isFirefox) {
+  browser.runtime.onInstalled.addListener(function (details) {
+    if (details.reason === 'install') {
+      browser.storage.session.set({ isFirstTimeInstall: true });
+    } else if (details.reason === 'update') {
+      browser.storage.session.set({ isFirstTimeInstall: false });
+    }
+  });
+} else if (!isManifestV3) {
+  browser.runtime.onInstalled.addListener(function (details) {
+    if (details.reason === 'install') {
+      global.localStorage.setItem('isFirstTimeInstall', true);
```

Comment:
Is `global.sessionStorage` available in Manifest V2 extensions?

Reply:
no, unfortunately it is not

Reply:
Hm, I've got an mv2 version installed and when I paste `global.sessionStorage` into the `background.html` console returns a `Storage` object:

![image](https://github.com/user-attachments/assets/b08b09d3-c2ef-4680-a391-1a8903b5061c)

Reply:
hmm, okay then. I was thinking of https://developer.chrome.com/docs/extensions/reference/api/storage#property-session, but I see that is different from `sessionStorage`

Reply:
Switched to using session storage in 45dda9f31d